app/Notifications/BaseNotification.php

```diff
@@ -4,12 +4,36 @@
 
 use Illuminate\Bus\Queueable;
 use Illuminate\Contracts\Queue\ShouldQueue;
-use Illuminate\Notifications\Messages\MailMessage;
 use Illuminate\Notifications\Notification;
 
-// Don't ShouldQueue yet - we're intentionally only doing this for admin notifications.
-class BaseNotification extends Notification
+abstract class BaseNotification extends Notification implements ShouldQueue
 {
+    use Queueable;
+
+    protected $arr;
+    protected $user;
+
+    public function __construct($arr, $user = null)
+    {
+        $this->arr = $arr;
+        $this->user = $user;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        if ($notifiable->invites == 1) {
```

Comment:
As I find the mismatch between the column name and what it actually means confusing whenever I see it - and now we only have to do it in one place - could you add a comment here: 

// If user being notified has opted in to receive emails.